src/screens/points/components/LeaderboardRow.tsx

```diff
@@ -20,6 +20,7 @@ import { useTheme } from '@/theme';
 import LinearGradient from 'react-native-linear-gradient';
 import { ButtonPressAnimation } from '@/components/animations';
 import { noop } from 'lodash';
+import useSwitchWallet from '@/hooks/useSwitchWallet';
```

Comment:
any reason to not include `useSwitchWallet` along with the other hooks in `hooks/index` ? (with line 6)

Reply:
no